README.md

```diff
@@ -66,7 +81,19 @@ This repository must include the following issue tags:
 
 **Team** and **Tutorial** tags are compulsory tags for each issue.
 
-### 2.2.4. Evaluation Repository
+### 2.2.4 Tester's Response Repository
+After the teams have responded to the bugs reported by their testers, the issues posted during that phase will be transferred over to this repository, with the identity of the poster anonymized.
+
+This repository will then be used by the application for individual tester to respond to the team's assessment of the bug they had reported initially.
+
+This repository must include the following issue tags:
+1. **Severity**: `severity.High`, `severity.Medium`, `severity.Low`
```

Comment:
Maybe we can exclude the categorical values over here and other areas too? Since they are bound to frequent changes.

Reply:
Sure 👍  Shall I remove the labels completely from the guide actually? Since our App automatically initializes them whenever it accesses a repository.

Reply:
Okay!